### PR TITLE
README for 0.1 branch.

### PR DESCRIPTION
**Pull request by cpovirk**

### CODE REVIEW

**Comment by cpovirk** [comment]
If this is way off base, I'd welcome a link to an example in which it's "done right," a doc of best practices, etc.

Thanks again for the recommendation.

**Comment by cpovirk** [comment]
(And if there are ways in which I got the branch itself wrong, etc., don't hesitate to let me know about that, too.)

File: README.md [hunk 1]
```diff
@@ -1,16 +1,23 @@
-# jspecify
+# jspecify 0.1
```
Comment (Stephan202) [inline README.md:1]: #116 is still open, but the consensus seems to move towards:
```suggestion
# JSpecify 0.1
```

File: README.md [hunk 2]
```diff
@@ -1,16 +1,23 @@
...
 
-**This project is a placeholder.** We are using the issue tracker for some of our conversations, but the existing code in the repo does not reflect our current design decisions, and we have many decisions left to make.
+(This README is for the 0.1 milestone. For the main project README, see
+[our GitHub landing page](https://github.com/jspecify/jspecify).)
 
-An artifact of well-named and well-specified annotations to power static
-analysis checks.
+This milestone will be tagged around August 19, 2020.
```
Comment (Stephan202) [inline README.md:6]: This sentence will look slightly awkward once tagging actually happened. Not an issue if one remembers to update it just prior to tagging, but safer might be to leave it out.

(What I guess we should do (post-tagging) is to link to the tag from the `main` README and have a sentence along these lines there. Something like "On August 19, 2020 we tagged release 0.1".)

File: README.md [hunk 3]
```diff
@@ -1,16 +1,23 @@
...
-This is not an officially supported product of any of the participant
-organizations.
+A frequent question has been: Why not just take what we have, remove the
+references to especially unsettled questions (like the existence of
+`@NullnessUnspecified`), and tag that? So that's what this milestone is for.
 
-## How to build
+This 0.1 milestone omits some of the work we've already completed, and we have
+much more work underway and yet to come. Everything remains **not set in
+stone**, and we don't expect tool authors or library owners to adopt 0.1.
```
Comment (Stephan202) [inline README.md:19]: :+1: Very clear disclaimer.

File: README.md (hunk 2 above)
Comment (cpovirk) [reply README.md:6]: Makes sense. And it occurs to me only now that GitHub will produce a list of releases for us, anyway.

**Comment by Stephan202** [comment]
W.r.t. the CLA warning: there's a CLA linked to the work email address I use for "regular" commits, but apparently there's no CLA linked to my personal/primary email address, which is the one Github uses for the PR code suggestion feature. Will look into linking the other address later today.

**Comment by Stephan202** [comment]
@googlebot I fixed it.

**Comment by Stephan202** [comment]
Looks like this repository has _two_ Google bots :smile:

**Comment by cpovirk** [comment]
Two Google bots, two Stephans... :)

I might have set up an extra one for the repo when one was already set up for the organization? I've removed the per-repo one. We'll see what happens.

**Comment by cpovirk** [comment]
And thanks for getting that taken care of. I was envisioning losing an afternoon over the casing of two letters :)

**Comment by cpovirk** [comment]
Thanks also for the comment on the branch name. I'm not sure I can predict at this point how we'll be navigating from here to 1.0. But it at least seems fair to say that we're not going to go back and make an emergency patch release of 0.1.1, so I don't see much need for a long-lived release-0.1 branch. I'll do the rename after I merge this.

**Comment by cpovirk** [comment]
https://github.com/jspecify/jspecify/tree/release-0.x

I think I did the branch rename in some suboptimal way, so I ended up wiping my local clone and my GitHub fork just to be safe.

**Comment by cpovirk** [comment]
I still got a googlebot response on https://github.com/jspecify/jspecify/pull/120, so my per-repo googlebot looks to have been redundant. I had technically just _disabled_ it earlier, so now I'll remove it for good.

**Pull request merged**